src/EncompassRest/Loans/Enums/HmdaCreditScoreForDecisionMaking.cs

```diff
@@ -0,0 +1,10 @@
+using System.Runtime.Serialization;
+
+namespace EncompassRest.Loans.Enums
+{
+    public enum HmdaCreditScoreForDecisionMaking
+    {
```

Comment:
review this : Schema issue with fields that have "allowed values" but are text entries.